src/theme/foundations/textStyles.ts

```diff
@@ -4,7 +4,7 @@ export const textStyles = {
     fontWeight: 700,
     fontSize: '2.75rem',
     lineHeight: '3.375rem',
-    letterSpacing: '0.05em',
+    letterSpacing: '0.04em',
```

Comment:
I don't see where we're using 0.04 for an `h1`
<img width="993" alt="image" src="https://user-images.githubusercontent.com/54227730/204620930-5c5436e6-4971-42ec-bbdc-6aae66cccec1.png">

Looks like we use 0.05 for the homepage h1's, and 0.03 for the other h1s